index.rst

```diff
@@ -1,61 +1,150 @@
+Small "development" runs will test out new Science Pipelines features.
+Larger "integration" runs will ensure that unusual data does not cause problems.
+Finally, a "mini-DRP" run, on the scale of 8% of the sky, will be executed to prove the data selection, configuration, and (human) processes.
+The Science Pipelines, System Performance, and Execution teams will meet throughout this process, under the leadership of the DRLCM.
+
+The Campaign Management system will be used for each of these smaller runs, allowing progress (and setbacks) to be monitored, reproduced, and tracked.
+
+Alert Production
+----------------
+
+Each night's Alert processing and daytime Solar System processing can be considered a campaign.
+It would similarly have a definition of the software and configurations to be used, and exclusion lists might be generated by the Observatory Operator (e.g. if it is known before processing begins that a certain image will be bad).
+Each visit causes an effective workflow submission, although the Alert Production execution framework may not use the Batch Production Service and workflow management at all.
+
+Commissioning
+-------------
+
+In Commissioning, both applying the same processing to different data and applying different processing to the same data will occur frequently.
+Being able to view the definitions of past campaigns and adjust the input datasets or pipeline configurations assists with this.
+
+
+The Scope
+=========
+
+The Campaign Management system sketched in the above use cases serves as a sort of electronic lab notebook, recording parameters and providing links to other systems that do the actual work of executing workflows or rendering plots.
+It is a system to manage campaigns, not one to execute them, although rudimentary execution capabilities (submit the next BPS command in sequence) can be added in later versions.
+Its primary interfaces are human-oriented, not machine-oriented.
+It is usable by staff, including developers, Commissioning staff, and Execution team members.
+It is not initially usable by science users, although with appropriate multi-user security and controls, it could be extended to them.
+
+The generation of inclusion/exclusion lists is acknowledged to require extreme flexibility, with information coming from any available source, including a random number generator.
+As a result, only a format for specifying the exclusion list is defined; the mechanisms and tools for generating such a list are left outside the scope of Campaign Management.
+In particular, the Exposure Log (LSE-490 :cite:`LSE-490` §3.2.4.2 and DMTN-173 :cite:`DMTN-173`) cannot be sufficient to generate inclusion/exclusion lists; it is but one source of information that goes into their generation.
+Inclusion/exclusion lists must be uploaded to the Butler Registry in order to enable joins with other tables and in order to maintain provenance.
```

Comment:
Provenance in terms of exactly what was the campaign definition (i.e., just a part of provenance)?  The provenance for what was generated in the QuantumGraphs and what was actually used (e.g., did a coadd code throw out an input due to some science value) should be saved independently from these lists.